### ISSUE TITLE
NTR: Reduced nicotinamide adenine dinucleotide (NADH) low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8404]
New term request Reduced nicotinamide adenine dinucleotide (NADH) low in blood (elthat_171215053239):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000406
- label: Diminished circulating reduced nicotinamide adenine dinucleotide (NADH) concentration
- parents: [Abnormal circulating purine concentration (HP:0004352)](https://hpo.jax.org/app/browse/term/HP:0004352)
- pmid: 
- def: The concentration of reduced nicotinamide adenine dinucleotide (NADH) in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8404
- synonyms:
